src/App/App.js

```diff
@@ -49,9 +49,14 @@ class App extends Component {
             revenue: data.movie.revenue,
             runtime: data.movie.runtime,
             tagline: data.movie.tagline,
-            averageRating: data.movie['average_rating']
+            averageRating: data.movie['average_rating'],
+            error: ''
           }})
         })
+        .catch(err =>{
```

Comment:
I forgot to add the catch in my last PR on this branch, so I figured this one could have a little bonus bit tacked on.

Reply:
Beautiful! I forgot too so I will send a similar thing to mine on next push :)